src/features/main_menu/_components/CCLog.vue

```diff
@@ -166,8 +164,8 @@ export default Vue.extend({
   align-self: stretch;
   display: inline-flex;
   height: inherit;
-  min-height: 95%;
-  max-height: 95%;
+  min-height: calc(100% - 24px);
+  max-height: calc(100% - 24px);
```

Comment:
These could optionally be `48px` instead of `24px`.  The current change means that the little arrow-caps at the bottom of the sidebars are always inline with the text, but they sometimes clip off and disappear when a new line is added at max height.  48px would have the arrow-caps 1 line above the bottom-most line but will keep them always visible.  I kept it at 24px because it looked better on Horus theme, where I think people would notice this the most.